src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs

```diff
@@ -291,8 +291,8 @@ darcsInstallPostInstall = unlines
 xmonadPostInstall :: String
 xmonadPostInstall = unlines
   [ "postInstall = ''"
-  , "  install -D man/xmonad.1 $doc/share/man/man1/xmonad.1"
-  , "  install -D man/xmonad.hs $doc/share/doc/$name/sample-xmonad.hs"
+  , "  install -D man/xmonad.1 ${!outputDoc}/share/man/man1/xmonad.1"
```

Comment:
You need two ticks for this to work like:

```
''${!outputDoc}
```

Otherwise Nix thinks it is a variable (as opposed to a bash variable)

Reply:
Ah of course, sorry for that @peti. I will PR the fix. I must have copy-pasted it incorrectly from the other nixpkgs editor buffer.